### DISCUSSION TITLE
UI change

### DISCUSSION THREAD

**Discussion by hweg2007** [UnchartedBull/OctoDash#3715] [Ideas]
I am thinking about upgrading my Ender-3 to octoprint w/ octodash and was wondering if you could possibly add a feature in settings so that you can change the fonts because I was planning on using DIN-1451 as the UI font. Thanks!

**Comment by hweg2007** [comment]
![image](https://github.com/UnchartedBull/OctoDash/assets/137019863/e79c3931-e429-4846-8084-e012417c4954)

**Reply by hweg2007** [reply]
I prefer this font because it is great for adding a modern look to just about anything.

**Reply by jneilliii** [reply]
You can use your own custom css and apply whatever font you want I suspect. Check out the [themes](https://github.com/UnchartedBull/OctoDash/tree/main/themes) folder for examples. I personally use the NOX theme and am very happy with it.
